sdk/cosmosdb/cosmos/test/internal/session.spec.ts

```diff
@@ -127,6 +127,44 @@ describe.skip("Session Token", function (this: Suite) {
     assert.ok(resource);
     assert.strictEqual(statusCode, 200);
   });
+
+describe("Integrated Cache Staleness", function (this: Suite) {
+  beforeEach(async function () {
+    await removeAllDatabases();
+
+  it.only("Should check if the max integrated cache staleness header is set and the value is correct.", async function () {
```

Comment:
This test case is inside the beforeEach block, is that intentional? Usually `it` blocks are outside `beforeEach`. Also please make sure to remove `.only` otherwise all other tests will be skipped

Reply:
fixed